uptime-kuma/templates/servicemonitor.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: {{ include "uptime-kuma.fullname" . }}  
+  labels:
+    {{ include "uptime-kuma.labels" . | nindent 4 }}
+spec:
+  endpoints:
+  - interval: 30s
+    path: /metrics
+    port: http
+    basicAuth:
+      password:
+        name: uptime-kuma-credentials
+        key: password
+      username:
+        name: uptime-kuma-credentials
+        key: username
+  namespaceSelector: # Scrape metrics from...
+    matchNames:
+    - {{ include "uptime-kuma.serviceMonitorNamespace" . }}
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: {{ include "uptime-kuma.fullname" . }}
```

Comment:
you seem to be adding an enable toggle to the `values` but it has no effect :wink: 
ideally we only provision the secret resource if the user opts to enable the monitor, WDYT?

*and please add the trim `-` in front of the label include*